embeddings/hybrid-search.ipynb

```diff
@@ -0,0 +1,1124 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "wDrRkb1v_LMi"
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2024 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pNK49xFFj8M1"
+   },
+   "source": [
+    "# Hybrid Search quickstart"
```

Comment:
I suggest adding a couple of things to this title:

1. The specific product(s) used in this notebook
2. A bit of info about the use case

Something along the lines of: "Combining Semantic & Keyword Search: A Hybrid Search Tutorial with Vertex AI Vector Search".

Reply:
+1, updating the title.